src/Symfony/Component/Ldap/Adapter/EntryManagerInterface.php

```diff
@@ -40,4 +40,13 @@ public function update(Entry $entry);
      * @param Entry $entry
      */
     public function remove(Entry $entry);
+
+    /**
+     * Renames an entry on the Ldap server.
+     *
+     * @param Entry  $entry
+     * @param string $newRdn
+     * @param bool   $removeOldRdn
+     */
+    public function rename(Entry $entry, $newRdn, $removeOldRdn = true);
```

Comment:
The readme states that the component is stable since Symfony 3.1 (https://github.com/symfony/ldap/blob/3.2/README.md). So I guess this is a BC break.